pytorch_lightning/strategies/ddp.py

```diff
@@ -135,6 +134,10 @@ def num_nodes(self, num_nodes: int) -> None:
         self._num_nodes = num_nodes
         self.set_world_ranks()
 
+    @property
+    def num_processes(self):
+        return len(self.parallel_devices) if self.parallel_devices is not None else 0
```

Comment:
there are some places in accelerator_connector where this is set manually? is that required?
for eg L898.

Reply:
same as above